.travis.yml

```diff
@@ -0,0 +1,12 @@
+dist: xenial
+language: go
+go: "1.13"
```

Comment:
I see that my PR had this version in `go.mod` probably added by IDE, but I'm not sure if we want to pin go version
Also, not sure if we have to pin go version to 1.12 like kubevirt has done for downstream purpose
Thoughts? @omeryahud @ksimon1

Reply:
I think we have a go-1.13 image downstream, I will check again

Reply:
Yes, there is a go-1.13 image downstream, this should be fine